server/src/main/java/com/linecorp/centraldogma/server/mirror/MirroringServicePluginConfig.java

```diff
@@ -68,38 +70,48 @@ public MirroringServicePluginConfig(
         this.maxNumBytesPerMirror = firstNonNull(maxNumBytesPerMirror, DEFAULT_MAX_NUM_BYTES_PER_MIRROR);
         checkArgument(this.maxNumBytesPerMirror > 0,
                       "maxNumBytesPerMirror: %s (expected: > 0)", this.maxNumBytesPerMirror);
+        this.zonePinned = zonePinned;
     }
 
     /**
      * Returns the number of mirroring threads.
      */
-    @JsonProperty
+    @JsonProperty("numMirroringThreads")
```

Comment:
revert?

Reply:
It looks like a boilerplate but I prefer this pattern because it is more deterministic. 
We also use a standard getter for boolean values for which `is` is dropped. I wasn't sure if that was intended or not.
https://github.com/line/centraldogma/blob/5f0a9a1eb7ed2db53db195cd5c8577d9bce8b459/server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaConfig.java#L500-L503

Anyway, it is unrelated to this PR so I will revert it.

Reply:
Yes, we're currently using it in a mixed manner, and it might be a good time to define a standard. 🤔